tests/installer/default.nix

```diff
@@ -43,14 +43,14 @@ let
   images = {
 
     /*
-    "ubuntu-14-04" = {
+      "ubuntu-14-04" = {
```

Comment:
The intendation changes here seem odd

Reply:
Oh whoops, that was the default autoformatter. I'll revert that.

Reply:
Fixed now.